bin/live/pycbc_live_plot_combined_single_fits

```diff
@@ -168,21 +182,23 @@ for ifo in ifos:
                                             c=bin_colour, linestyle=':',
                                             label=count_lab))
 
-    ax_alpha.set_xlabel('Days since ' + start_date)
     ax_alpha.set_ylabel('Fit coefficient')
-    ax_alpha.set_xlim([ad[0], ad[-1]])
+    ax_alpha.set_xticks((xtix - min_start) / 86400)
```

Comment:
I am a bit confused by this, what guarantees that a time offset divided by 86400 will fall exactly at midnight? Or do we not care that ticks are at midnight?